core/src/components/textarea/textarea.tsx

```diff
@@ -189,7 +189,9 @@ export class Textarea implements ComponentInterface {
   }
 
   componentDidLoad() {
-    this.runAutoGrow();
+    setTimeout(() => {
+      this.runAutoGrow();
+    });
```

Comment:
```suggestion
    raf(() => this.runAutoGrow());
```

Let's use the `raf` helper instead. This will use `requestAnimationFrame` instead of `setTimeout` and will also avoid NgZone being run.

You can modify the import line at the top of the file: 

`import { debounceEvent, findItemLabel, raf } from '../../utils/helpers';`

Reply:
Thanks for the feedback. Changes made!

Reply:
Hi Guys, I have a project where I am getting this problem. How do I implement this fix?

Reply:
This has been merged and is in the current 5.1.0 release